sites/fast-website/src/public/index.html

```diff
@@ -65,7 +65,47 @@
         <!-- frameworks:start -->
         <section class="section">
           <div class="section-component">
-            Framework components
+            <site-content-placement divider=true >
```

Comment:
```suggestion
            <site-content-placement divider>
```

For Boolean attributes you don't need to assign a value - you can just add the attribute name :)

Reply:
Awesome! Thank you.